src/components/slider/slider.component.ts

```diff
@@ -32,6 +32,8 @@ export class SliderComponent implements OnInit, AfterViewInit, DoCheck, OnDestro
     sliderThumb = SliderThumb;
     sliderTickType = SliderTickType;
     sliderThumbEvent = SliderThumbEvent;
+    lowerDragging = false;
```

Comment:
Shouldn't need to add these variables. The dragging status should already be available in the thumbs object. eg. `thumbs.lower.drag` & `thumbs.upper.drag`